avaframe/runScripts/runStandardTestsCom1DFAOrig.py

```diff
@@ -108,14 +106,19 @@
     plotDict = oP.plotAllPeakFields(avaDir, cfgMain['FLAGS'], modName)
 
     # Set directory for report
-    reportDir = os.path.join(avaDir, 'Outputs', modName, 'reports')
+    reportDir = pathlib.Path(avaDir, 'Outputs', modName, 'reports')
```

Comment:
Would it make sense to convert avaDir to a pathlib object right at the beginning so that most of these lines can be converted to `avaDir / Outputs / ....` ?